orttraining/orttraining/models/bert/main.cc

```diff
@@ -367,6 +361,15 @@ Status ParseArguments(int argc, char* argv[], BertParameters& params, OrtParamet
     ORT_RETURN_IF_NOT(params.data_parallel_size > 0, "data_parallel_size must > 0");
     ORT_RETURN_IF_NOT(params.horizontal_parallel_size > 0, "horizontal_parallel_size must > 0");
 
+    int seed = flags["seed"].as<int>();
```

Comment:
should it be int64_t?

Reply:
Seems reasonable.  I've switched these to int64_t.